### DISCUSSION TITLE
How to save model for ONNX and Keras after bentoml 1.4

### DISCUSSION THREAD

**Discussion by cceasy** [bentoml/BentoML#5048] [Q&A]
Hi @frostming  I noticed that onnx and keras were marked as deprecated, and there is no alternatives in the log, I also check the office document, no alternatives found. Can you please tell me how to handle onnx and keras models, thank you in advance.
https://github.com/bentoml/BentoML/blob/v1.3.10/src/bentoml/__init__.py#L178-L189

**Comment by aarnphm** [comment]
check `bentoml.models.create`

**Comment by aayostem** [comment]
Modern Keras is part of TensorFlow, so it's better to use bentoml.tensorflow
ONNX: Runtime integration was limited; better approaches exist now. For instance
    -  you can use ONNX Runtime Directly
    - or you can migrate ONNX to TensorFlow/PyTorch
